dist/angular-bootstrap-toggle.js

```diff
@@ -277,7 +294,7 @@
 						if (self.disabled) { // prevent changing .$viewValue if .disabled == true
 							return false;
 						} else {
-							self.ngModelCtrl.$setViewValue(!self.ngModelCtrl.$viewValue);
+							self.ngModelCtrl.$setViewValue(!angular.equals(self.ngModelCtrl.$viewValue, trueValue) ? trueValue : falseValue);
```

Comment:
Same here: jshint is complaining the line is too long, which breaks the build.

Reply:
Fixed.